src/utils/unique-id.js

```diff
@@ -2,6 +2,6 @@ let index = 0;
 
 export default {
   generate() {
-    return 'mui-id-' + (index++);
+    return `mui-id-${index++}`;
```

Comment:
Does it not work? Honestly I think the few times we use this utility I wonder if it's even worth it :grin: 

If string templating is causing issues in this isolated case, I'm still in favor of merging this, even if we have to add `//eslint-disable-line` to this